tests/telemetry/events/dhcp_relay_events.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+import logging
+import time
+from run_events_test import run_test
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+tag = "sonic-events-dhcp-relay"
+
+interface = ""
+ipv4_address = ""
+ipv6_address = ""
+
+
+def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
+    logger.info("Beginning to test dhcp-relay events")
+    try:
+        backup_ip_address(duthost)
+    except Exception:
+        logger.info("Skipping test, no Vlan interface")
+        return
+
+    try:
+        run_test(duthost, gnxi_path, ptfhost, data_dir, validate_yang, invoke_dhcp_relay_bind_failure,
+                 "dhcp_relay_bind_failure.json", "sonic-events-dhcp-relay:dhcp-relay-bind-failure", tag)
+    finally:
+        restart_dhcp_container(duthost)
+
+
+def backup_ip_address(duthost):
+    # grab ip_addresses with subnet mask
+    interface_facts = duthost.show_ip_interface()['ansible_facts']['ip_interfaces']
+    global interface
+    interface = next((key for key in interface_facts.keys() if 'Vlan' in key), None)
```

Comment:
>interface [](http://example.com/codeflow?start=4&length=9)

Could you add some code comment to explain how you pick one interface?

Reply:
Added comment above